friends/src/App.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+import { ToastContainer } from 'react-toastify'
+import "react-toastify/dist/ReactToastify.min.css";
+import FriendsList from './components/FriendsList/FriendsList'
+import './App.css';
+import Login from './components/Login/Login';
+import PrivateRoute from './components/PrivateRoute'
+import { Switch } from 'react-router-dom'
+import FriendForm from './components/FriendForm/FriendForm';
+import NavBar from './components/NavBar';
+
+function App() {
+  return (
+    <div className="App">
+      {localStorage.getItem('token') && <NavBar/>}
```

Comment:
This line make the NavBar invisible if the token is not present. Whenever a user reach your page what he sees is an empty page.